src/test/java/no/unit/nva/cristin/projects/FetchOneCristinProjectTest.java

```diff
@@ -109,15 +110,15 @@ void handlerReturnsBadGatewayWhenBackendThrowsIoException() throws Exception {
     }
 
     @Test
-    void handlerReturnsServerErrorExceptionWhenBackendThrowsUriSyntaxException() throws Exception {
+    void handlerReturnsServerErrorExceptionWhenBackendThrowsGenericException() throws Exception {
         cristinApiClientStub = spy(cristinApiClientStub);
 
-        doThrow(URISyntaxException.class).when(cristinApiClientStub).getProject(any(), any());
+        doThrow(RuntimeException.class).when(cristinApiClientStub).getProject(any(), any());
         handler = new FetchOneCristinProject(cristinApiClientStub, environment);
-        GatewayResponse<NvaProject> nextResponse = sendQueryWithId(DEFAULT_ID);
+        GatewayResponse<NvaProject> response = sendQueryWithId(DEFAULT_ID);
 
-        assertEquals(HttpURLConnection.HTTP_INTERNAL_ERROR, nextResponse.getStatusCode());
-        assertEquals(APPLICATION_PROBLEM_JSON, nextResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        assertEquals(HttpURLConnection.HTTP_INTERNAL_ERROR, response.getStatusCode());
```

Comment:
What is the expected message to the api-client?

Reply:
Should i add an expected message to HTTP_INTERNAL_ERROR exceptions only or add them to all relevant tests?

Reply:
leave it to next PR